### ISSUE TITLE
Use variables inside strings/links

### ISSUE THREAD

**Issue by malua** [ionic-team/trapeze#112]
Hey, 
first of all thanks for this cool project.

I have this config file:

```
DEEPLINK_PROTOCOL:
    default: classiccard

....

 entitlements:
          replace: true
          entries:
            - aps-environment: development
            - com.apple.developer.associated-domains:
                [
                  applinks:$DEEPLINK_PROTOCOL.app.link,
                  applinks:$DEEPLINK_PROTOCOL-alternate.app.link,
                  applinks:$DEEPLINK_PROTOCOL.test.app.link,
                  applinks:$DEEPLINK_PROTOCOL.test-alternate.app.link,
                ]
```

This is what's generated:
```
<key>com.apple.developer.associated-domains</key>
<array>
    <string>applinks:</string>
    <string>applinks:classiccard-alternate.app.link</string>
    <string>applinks:</string>
    <string>applinks:-alternate.app.link</string>
</array>
```

Unfortunately trapeze is only replacing the second line correctly.
Could this be related to this PR https://github.com/ionic-team/trapeze/issues/43?

 
Is there a way to tell trapeze how to parse/replace this correctly? 

All the best and thanks, 
Lukas

**Comment by mlynch** [comment]
I think you're right this is related to #43. In fact, I don't know why I merged that as I don't think periods should be allowed in var names (for example, they aren't allowed in bash). I'll take another look at this

**Comment by mlynch** [comment]
Reverted in 30c98cc1cac7bfd2cda5e1baa2ae1660fc218a9e

**Comment by mlynch** [comment]
Fixed in 5.0.7

**Comment by malua** [comment]
Thank you!
